lib/fog/ovirt/core.rb

```diff
@@ -10,6 +10,16 @@ module Errors
       class ServiceError < Fog::Errors::Error; end
       class SecurityError < ServiceError; end
       class NotFound < ServiceError; end
+      class OvirtError < Fog::Errors::Error; end
+
+      class OvirtClientError < OvirtError
```

Comment:
This is a little confusing, `OvirtClient` does not show this is an actual error we received from Ovirt.
Maybe this can be renamed to `OvirtEngineError`?
My first impression was that `OvirtError` came from ovirt so I didn't understand it was the base class.

Reply:
here is a rename, tested it as well.